### DISCUSSION TITLE
How to make external links with Markdown?

### DISCUSSION THREAD

**Discussion by i-do** [datenstrom/community#570] [Ask a question] [answered]
Suddenly find that markdown {target="_blank"} doesn't work any more.
I often use this add on my site.

`
[Text Title](url){target="_blank"}
`

Does somebody knows why?

**Add:** I've find [Pasedown Extra extention](https://github.com/erusev/parsedown-extra). 
Will try to fix it in the evening.

**Comment by markseuffert** [comment, marked as answer]
The following works in the [demo page](https://datenstrom.se/yellow/demo/), Markdown and HTML: 

`[Open link in new tab](https://datenstrom.se){target=_blank}.`
`<a href="https://datenstrom.se" target="_blank">Open link in new tab</a>.`

For more information see:
https://github.com/annaesvensson/yellow-markdown
https://github.com/annaesvensson/yellow-parsedown
